src/common/indexedDB/indexedDB.ts

```diff
@@ -49,12 +49,14 @@ export interface IndexedDBAPI {
  */
 export class IndexedDBUtil implements IndexedDBAPI {
     private store;
+    private readonly dbName: string = 'default-db';
+    private readonly storeName: string = 'default-store';
 
     constructor() {
         if (!window.indexedDB) {
             throw 'IndexedDB is not available';
         }
-        this.store = new Store();
+        this.store = new Store(this.dbName, this.storeName);
```

Comment:
We should be injecting this dependency rather than making this component responsible for it.

Reply:
Good suggestion. Changed.